examples/blog-standalone/mkdocs.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2016-2023 Martin Donath <martin.donath@squidfunk.com>
+#                         Alex Voss <alex@corealisation.com>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+# -----------------------------------------------------------------------------
+# Recommended: set up configuration validation, see https://t.ly/xpZXU
+# -----------------------------------------------------------------------------
+
+# Project information
+site_name: Standalone blog example
+
+# Theme
+theme:
+  name: material
+  features:
+  - navigation.tabs
+  - navigation.indexes
+
+# Plugins
+plugins:
+  - search
+  - blog:
+      blog_dir: demo
```

Comment:
Why is the blog contained in the `demo` directory? IMHO, by definition, a standalone blog lives at the root 😅

Reply:
Right, I was going to ask. The `demo/` directory is not my invention but something I copied from earlier examples. The way I made sense of it was that it puts `README.md` in the `docs/` folder as the main page for the example, which may be doing things that have nothing to do with the example itself, such as showing bits of the code used.

Reply:
Hmm, maybe we should really move this into a new section called recipes, cookbooks, or whatever, then. We should definitely keep the demo for examples, but this doesn't really work for the standalone blog. The standalone blog could then also feature a full-blown setup with tags, social cards, etc., basically everything that's in the free version.´, and we could slowly extend it to be a good starter. What do you think? 

Maybe we should call it starters, albeit that makes me hungry.

Reply:
> feature a full-blown setup with tags, social cards, etc.

That sounds good to me. I will start working on this. TBH, I always felt that the examples are a bit too minimalistic to be of much help to many people but I have no empirical evidence to back that up. Definitely happy to do more to show what amazing things are possible if you put things together in the right way. I keep doing bits of work now and then on the tutorial (recipe, starter, entrée, smorgasbord...) and, no doubt will add more as I pick up useful things while looking at the blog functionality. 

One question, though: should we publish the blog-standalone example even if it is a bit odd? We could mention that in the README.

Another one: I have started to write a contributors.md file to document how the examples work. Put a note on us not accepting community contributions just now. If you don't like publishing it (yet) then I could stick the text in to the internal repo for now so we can at least look at it and refine it?

Reply:
One more thing: if you're on the contributing.md, it would be perfect if you could find some time to work on https://github.com/squidfunk/mkdocs-material/issues/6153, which we're desperately missing in our docs. I guess there might be some overlap.